python/hsfs/feature_view.py

```diff
@@ -2128,6 +2124,303 @@ def delete_all_training_datasets(self):
+    ) -> List["fmr.FeatureMonitoringResult"]:
+        """Fetch feature monitoring history for a given feature monitoring config.
+
+        !!! example
+            ```python3
+            # fetch your feature view
+            fv = fs.get_feature_view(name="my_feature_group", version=1)
+
+            # fetch feature monitoring history for a given feature monitoring config
+            fm_history = fv._get_feature_monitoring_history(
+                config_name="my_config",
+                start_time="2020-01-01",
+            )
+
+            # or use the config id
+            fm_history = fv._get_feature_monitoring_history(
+                config_id=1,
+                start_time=datetime.now() - timedelta(weeks=2),
+                end_time=datetime.now() - timedelta(weeks=1),
+                with_statistics=False,
+            )
+            ```
+
+        # Arguments
+            config_name: The name of the feature monitoring config to fetch history for.
+                Defaults to None.
+            config_id: The id of the feature monitoring config to fetch history for.
+                Defaults to None.
+            start_date: The start date of the feature monitoring history to fetch.
+                Defaults to None.
+            end_date: The end date of the feature monitoring history to fetch.
+                Defaults to None.
+            with_statistics: Whether to include statistics in the feature monitoring history.
+                Defaults to True. If False, only metadata about the monitoring will be fetched.
+
+        # Raises
+            `hsfs.client.exceptions.RestAPIError`.
+            `hsfs.client.exceptions.FeatureStoreException`.
+            ValueError: if both config_name and config_id are provided.
+            TypeError: if config_name or config_id are not respectively string, int or None.
+
+        # Return
+            List[`FeatureMonitoringResult`]
+                A list of feature monitoring results containing the monitoring metadata
+                as well as the computed statistics for the detection and reference window
+                if requested.
+        """
+        if not self._id:
+            raise FeatureStoreException(
+                "Only Feature View registered with Hopsworks can fetch feature monitoring history."
+            )
+
+        fm_result_engine = (
+            feature_monitoring_result_engine.FeatureMonitoringResultEngine(
+                feature_store_id=self._featurestore_id,
+                feature_view_id=self._id,
+                feature_view_name=self._name,
+                feature_view_version=self._version,
+            )
+        )
+
+        return fm_result_engine.get_feature_monitoring_results(
+            config_name=config_name,
+            config_id=config_id,
+            start_time=start_time,
+            end_time=end_time,
+            with_statistics=with_statistics,
+        )
+
+    def _enable_scheduled_statistics_monitoring_fluent(
```

Comment:
The method name does not match what it really does. The purpose of the method is `create_fm_config`.

Reply:
We could rename both methods from enable to create